src/Database/Type/EnumType.php

```diff
@@ -167,7 +167,7 @@ public function toStatement(mixed $value, Driver $driver): int
      */
     public function marshal(mixed $value): ?BackedEnum
     {
-        if ($value === null || $value === '') {
+        if ($value === null) {
```

Comment:
I am fairly certain we added some of this for compatibility with int backed enums and empty values 
Will have to check that there arent regressions in that direction

Reply:
Empty string to int backed enum is still accounted for (check line 187-190) and returns `null` as before. There are also tests for the same.